src/iosMain/kotlin/com/jakewharton/byteunits/BinaryByteUnit.kt

```diff
@@ -0,0 +1,162 @@
+package com.jakewharton.byteunits
+
+import platform.Foundation.NSNumber
+import platform.Foundation.NSNumberFormatter
+
+/**
+ * A [BinaryByteUnit] represents power-of-two byte sizes at a given unit of granularity and
```

Comment:
Do we want to duplicate the documentation or only have it at the commonMain level?